lib/block-supports/index.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Block support flags.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Filter the registered blocks to apply the block supports attributes registration.
+ */
+function gutenberg_register_block_supports() {
+	$block_registry         = WP_Block_Type_Registry::get_instance();
+	$registered_block_types = $block_registry->get_all_registered();
+	// Ideally we need a hook to extend the block registration
+	// instead of mutating the block type.
+	foreach ( $registered_block_types as $block_type ) {
+		gutenberg_register_alignment_support( $block_type );
+	}
+}
+
+add_action( 'init', 'gutenberg_register_block_supports', 21 );
```

Comment:
This seems to fix it!  

However, I find it interesting that decreasing the priority from 11 to 21 solved the issue.  Since the docs note `Lower numbers correspond with earlier execution`, my first assumption would have been that decreasing the value to have this registered sooner would have helped.  Any ideas why decreasing the priority helped here?

Reply:
This should run after block registration as it alters it,  but by looking at the code,  block registration seems to run on the default priority which is 10. So in theory 11 or 21 should have similar results but for some reason, I don't understand yet, it seems registrations is done later.